ortsbegehung.md

```diff
@@ -9,8 +9,12 @@
 - [ ] Taschenlampe
 - [ ] Handschuhe
 - [ ] Maßband
+
+## Selbstschutz
+
 - [ ] ggf. Helm
 - [ ] ggf. Hörschutz/Ohrstöpsel (z.B. auf Kirchtürmen)
+- [ ] Handschuhe, Atemschutz, Einweganzug (z.B. gegen Taubendreck)
```

Comment:
Handschuhe werden bereits in Zeile 10 erwähnt.

Wenn wir Gegenstände einzeln auflisten, kann man sie wirklich abhaken.

Vielleicht sollten wir eine Datei "Schutzausrüstung" anlegen?